bundles/org.eclipse.mita.target/oxygen.target

```diff
@@ -40,5 +40,9 @@
 <unit id="org.eclipse.launchbar.feature.group" version="2.2.0.201711162100"/>
 <repository location="http://download.eclipse.org/releases/oxygen"/>
 </location>
+<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
```

Comment:
Arduino development is actually supported in CDT proper, so this might be a better option, right? https://marketplace.eclipse.org/content/eclipse-c-ide-arduino